m4_sessions/1710-inning/group_four.md

```diff
@@ -1,10 +1,31 @@
-Group Four
+****Group Four****
+
+**Members**
+- David Becker
+- Jon Beckman
+- Sam Johnson
+- Yohanan Assefa
+- Michael Sagopolutele
+- Daniel Vogelsang
+
+Session Title: Our new role in social movements developers
+Description: Why should students come to this session?
 
-Session Title: 
-Description: Why should students come to this session? 
 
 Session Outcomes: What will students leave with or gain from this session?
 
+
 Session Outline:  Include the flow of the session and the time for each section.
 
-Names of group members 
+**Intro**
+
+- Conversation
+- Diary
+- Examples of social movements and the role of technology.
```

Comment:
It might be worth leading with the examples--getting people to think of the role that tech has played (for success and failure of movements)

Reply:
@emhickmann We discussed this earlier this week and decided to focus our time on positive examples - although we may at least make mention of the fact that there is obviously a number of counterexamples (Brexit, 2016 election, etc).  We have this more flushed out on our local branch and will push up a more finalized/formal outline today after we meet!  Thank you!